m/processing_tools/preprocessGliderData.m

```diff
@@ -739,6 +757,9 @@
     if isfield(ctd_choice_list, 'time')
       time_ctd_field = ctd_choice.time;
     end
+    if isfield(ctd_choice_list, 'oxygen') % -FC
+      oxygen_ctd_field = ctd_choice.oxygen;
+    end    
```

Comment:
There is not need to sign the changes. That is the job of the version control system
(`git blame` the file from command line, or blame button in the file's view on Github).